### PR TITLE
Added prathameshborse as mentor

### PR DESCRIPTION
**Pull request by prathamesh-borse**
Added Prathamesh Borse as a mentor.

### CODE REVIEW

**Comment by thisisamank** [comment]
Hi @prathamesh-borse the formatting is disturbed, can you check?

File: MENTORS.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-| 21. | Riddhi Suteri | 🐦[Twitter](https://twitter.com/ridsuteri) <br/> 🎓 [LinkedIn](https://www.linkedin.com/in/riddhi-suteri/) | [@ridsuteri](https://github.com/ridsuteri) | Full Stack Web Developer, Mentor @ GirlScript Foundation |
-| 22. | Deepak Verma | 🐦[Twitter](https://twitter.com/deeqakkk) <br/> 🎓 [LinkedIn](https://www.linkedin.com/in/deeqakkk/) | [@deeqakkk](https://github.com/deeqakkk) | Full Stack Web Developer, Mentor @ GWOC'21 |
-| 23. | Kanak | 🐦[Twitter](https://twitter.com/kanak2212?s=90) <br/> 🎓 [LinkedIn](https://www.linkedin.com/in/kanak-a21384186/) | [@kanak22](https://github.com/kanak22) | Full Stack Web Developer and open-source enthusiast |
-| 24. | Saifur Rahman Mohsin | 🐦[Twitter](https://twitter.com/SaifurMohsin) <br/> 🎓 [LinkedIn](https://www.linkedin.com/in/saifurrahmanmohsin/) | [@mohsin](https://github.com/mohsin) | Freelance Fullstack Developer |
+| Serial No. | Name :medal_military:   | Social Media :wave:                                                                                                                 | GitHub :octocat:                                           | About                                                    |
```
Comment (thisisamank) [inline MENTORS.md:2]: Do not change already present content.

**Review by thisisamank — CHANGES_REQUESTED** [review]

**Comment by prathamesh-borse** [comment]
I am Closing this Pull request and sending a new pull request.

**Comment by prathamesh-borse** [comment]
I am saving that mentors.md file then it changes automatically what should i do?

**Pull request closed without merging**